include/tvm/script/ir_builder/relax/ir.h

```diff
@@ -160,13 +126,12 @@ TVM_DLL Optional<tvm::relax::Var> EmitMatchShape(const tvm::relax::Expr& value,
 /*!
  * \brief Annotate and check the type and shape of relax var.
  * \param var The input var to be annotated.
- * \param anno_type The annotated type.
- * \param anno_shape The annotated shape, which can be undefined.
+ * \param anno_struct_info The annotated struct info, which can be undefined.
  * \note This function will check if the type of var is compatible with the annotated type.
  * And we annotate to the var with more detailed type.
  */
-TVM_DLL void AnnotateTypeShape(const tvm::relax::Var& var, const Type& anno_type,
-                               const Optional<tvm::relax::Expr>& anno_shape);
+TVM_DLL void AnnotateTypeShape(const tvm::relax::Var& var,
```

Comment:
Needs some renaming on the use-sites as well
```suggestion
TVM_DLL void AnnotateStructInfo(const tvm::relax::Var& var,
```